maven-release-manager/src/test/resources/projects/rewrite-for-release/pom-with-parent-and-cifriendly-expressions/expected-pom.xml

```diff
@@ -31,6 +31,8 @@
 
   <properties>
     <revision>1.0</revision>
+    <sha1>.123</sha1>
+    <changelist></changelist>
```

Comment:
I think it would be nice if this PR would instead produce `<changelist />`, as that is the coding style used in Maven's own POM (which has, e.g. `<relativePath />`).